api/users/usersRouter.js

```diff
@@ -2,18 +2,25 @@ const usersRouter = require("express").Router();
 
 const { validate, objectIdValidator } = require("./middlewares");
 const {
-  getAllUsersSchema,
+  allUsersSchema,
   newUserSchema,
-  updateUserSchema
+  updateUserSchema,
+  deleteUserSchema
 } = require("./schemas");
 const controllers = require("./controllers");
 
-usersRouter.get("/", validate(getAllUsersSchema), controllers.listUsers);
+usersRouter.get("/", validate(allUsersSchema), controllers.listUsers);
 
 usersRouter.post("/", validate(newUserSchema), controllers.createUser);
 
 usersRouter.use("/:userId", objectIdValidator("userId"));
 
 usersRouter.put("/:userId", validate(updateUserSchema), controllers.updateUser);
 
+usersRouter.patch(
+  "/:userId/isDeleted",
+  validate(deleteUserSchema),
+  controllers.updateUser
+);
```

Comment:
You should use DELETE method here.
No need for `isDeleted ` in url in this case

Reply:
I've fixed problems already. See commit below.